packages/format-library/src/bold/index.js

```diff
@@ -20,11 +20,11 @@ export const bold = {
 	className: null,
 	edit( { isActive, value, onChange, onFocus } ) {
 		function onToggle() {
-			onChange( toggleFormat( value, { type: name } ) );
+			onChange( toggleFormat( value, { type: name }, title ) );
```

Comment:
What do you think about moving `title` into the `RichTextFormat` object? That way we're not adding an extra positional argument. Positional arguments make it more difficult in the future to change the API without breaking backwards compatibility. 

```js
onChange( toggleFormat( value, { type: name, title } ) );
```